app/assets/stylesheets/mixins/_grid-clearfix.scss

```diff
@@ -0,0 +1,25 @@
+// Provides an easy way to include a clearfix for containing floats.
+// Accepts options of `top`, `bottom` and `both`.
+
+@mixin grid-clearfix($location: bottom) {
```

Comment:
This confusing me…the `$location` argument. I’m not sure I know why, when clearfixing, I would need to specify a location of something?